src/coreclr/src/vm/dynamicmethod.cpp

```diff
@@ -1197,6 +1197,25 @@ LCGMethodResolver::IsValidStringRef(mdToken metaTok)
     return GetStringLiteral(metaTok) != NULL;
 }
 
+LPCWSTR
+LCGMethodResolver::GetStringLiteral(mdToken metaTok, int* length)
```

Comment:
It would be better to call this method `GetStringLiteralLength` and just make it return length.